<?xml version="1.0" ?>
<testCase duration="0.066" fail="1" name="SET-222" pass="0" startdate="20160215" starttime="150613">
	<block fail="0" pass="0"/>
	<block fail="0" pass="0"/>
	<block fail="1" pass="0">
		<block fail="1" pass="0">
			<call fail="1" pass="0">functions.raiseError('Intentional exception in execution phase')</call>
		</block>
		<block fail="0" pass="1">
			<call fail="0" pass="1">functions.addValues(vars.a, &quot;C&quot;)</call>
		</block>
		<block fail="0" pass="0">
			<call fail="0" pass="1">functions.addValues(vars.a, &quot;A&quot;)</call>
		</block>
	</block>
	<block fail="1" pass="0">
		<call fail="1" pass="0">functions.compareValues(vars.a, &quot;ABC&quot;)</call>
	</block>
</testCase>
